samples/bluetooth/le_audio/broadcast_source/README.rst

```diff
@@ -0,0 +1,68 @@
+.. _bluetooth-broadcast-source-sample:
+
+BLE Audio Broadcast Source Sample
+#################################
+
+Overview
+********
+
+This sample demonstrates the LE audio broadcast source use-case.
+
+The broadcast name used can be configured using `CONFIG_BROADCAST_NAME`.
+This might need to be changed for compatibility with a 3rd party broadcast sink device, as the broadcast name is sometimes used to choose which broadcast source to receive from.
+
+Building and Running
+********************
+
+This sample can be found under :zephyr_file:`samples/bluetooth/broadcast_source` in the
+sdk-alif tree.
+
+See :ref:`Alif bluetooth samples section <alif-bluetooth-samples>` for details.
+
+Configuration options:
+
+.. list-table::
+
+    * - :file:`prj.conf`
+      - This is the standard default config. Sampling frequency 48kHz, 10ms frame duration, 100 octets per codec frame. The default retransmissions are set to 4.
+
+    * - :file:`overlay-auracast.conf`
```

Comment:
Please update overlays and add one example build overlay.

Reply:
Overlay information updated. Zephyr's own documentation already shows how an overlay file can be applied.